### PR TITLE
Add missing namespace to HCO CR manifest

### PR DESCRIPTION
**Pull request by rwsu**

### CODE REVIEW

**Comment by rwsu** [comment]
kubevirt-hyperconverged appears to be the required namespace. I tried deploying to another namespace and it didn't work.

File: deploy/converged/crds/hco.cr.yaml [hunk 1]
```diff
@@ -3,4 +3,5 @@ apiVersion: hco.kubevirt.io/v1alpha1
 kind: HyperConverged
 metadata:
   name: hyperconverged-cluster
+  namespace: kubevirt-hyperconverged
```
[the remark this answers is not in the document]
Comment (rwsu) [reply deploy/converged/crds/hco.cr.yaml]: Done. Thanks.

**Comment by djzager** [comment]
If you run `./hack/build-manifests.sh` and include the changes then the rendered templates should be updated.

**Comment by ohadlevy** [comment]
whats the status of this pr? thanks.

**Comment by djzager** [comment]
just needs to run `./hack/build-manifests.sh` to update the generated CSV(s) + other manifests. @rwsu will likely need to rebase to avoid conflicts with master.

**Pull request closed without merging**